edx_api/course_list/fixtures/course_list_paginated.json

```diff
@@ -0,0 +1,40 @@
+{
+    "count": 3,
+    "next": "http://192.168.33.10:8000/api/courses/v1/courses/?page=2",
+    "previous": null,
+    "num_pages": 2,
```

Comment:
I don't think these are available in API response at this level. I see them inside pagination key only.